.github/ISSUE_TEMPLATE/failing-tests.md

```
@@ -0,0 +1,39 @@
+---
+name: Failing Test / Acceptance Test Failure
+about: Report a failing acceptance test in the Janssen Terraform provider
+title: '[TEST] '
+labels: test-failure, terraform
+assignees: ''
+
+---
+
+## Failing Test
+
+### Description
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)